lib/page/set-errors/set-errors.js

```diff
@@ -136,10 +138,12 @@ external.getFormattedError = (position, controlInstance, errorType, error, lang)
       // eg. error.required
       `error.${errorType}`
     ]
+
     errorLookups.some(errorLookup => {
       errorString = external.getString(errorLookup, lang)
-      return errorString
+      return !!errorString
```

Comment:
Truthiness is implicit in the original statement, but it doesn't hurt to make it explicit for clarity